cibuildwheel/logger.py

```diff
@@ -120,19 +162,22 @@ def build_start(self, identifier: str) -> None:
         self.build_start_time = time.time()
         self.active_build_identifier = identifier
 
-    def build_end(self) -> None:
+    def build_end(self, filename: Path | None) -> None:
         assert self.build_start_time is not None
         assert self.active_build_identifier is not None
         self.step_end()
 
         c = self.colors
         s = self.symbols
         duration = time.time() - self.build_start_time
+        duration_str = humanize.naturaldelta(duration, minimum_unit="milliseconds")
 
         print()
-        print(
-            f"{c.green}{s.done} {c.end}{self.active_build_identifier} finished in {duration:.2f}s"
+        print(f"{c.green}{s.done} {c.end}{self.active_build_identifier} finished in {duration_str}")
```

Comment:
<img width="524" height="345" alt="image" src="https://github.com/user-attachments/assets/0e4d577b-ca09-4c2f-9d0c-5fb0642e1eb4" />

I'm not sure about some of these uses of naturaldelta et al. They aren't as readable/scannable as the numerical forms IMO.

Reply:
You can use the minimum size setting to avoid the "a moment" ones